src/zos-io.cc

```diff
@@ -700,6 +714,62 @@ int __disableautocvt(int fd) {
   return fcntl(fd, F_CONTROL_CVT, &req);
 }
 
+int __ccsid_new_file() {
+  char* encode_file_new = getenv("_ENCODE_FILE_NEW");
+
+  int ccsid = 819;
+
+  if (encode_file_new)
+    if (strcmp(encode_file_new, "IBM-1047") == 0) {
+      ccsid = 1047;
+    } else if (strcmp(encode_file_new, "BINARY") == 0) {
+      // Set the file descriptor to binary mode
+      ccsid = FT_BINARY;
+    }
+
+  return ccsid;
+}
+
+int __tag_new_file_filename(int fd, std::string filename) {
+
+ auto it = existing_files.find(filename);
+ if(it != existing_files.end())
```

Comment:
```suggestion
 if (it != existing_files.end()) {
```
For style consistency, curly brace is at the end of the line